qiskit/synthesis/two_qubit/two_qubit_decompose.py

```diff
@@ -571,11 +570,10 @@ def specialize(self):
 
 
 class TwoQubitWeylControlledEquiv(TwoQubitWeylDecomposition):
-    """U ~ Ud(α, 0, 0) ~ Ctrl-U
-
+    r""":math:`U \sim U_d(\alpha, 0, 0) \sim \text{Ctrl-U}`
     This gate binds 4 parameters, we make it canonical by setting:
-        K2l = Ry(θl).Rx(λl) ,
-        K2r = Ry(θr).Rx(λr) .
+        :math:`K2_l = Ry(\theta_l).Rx(\lambda_l)` ,
+        :math:`K2_r = Ry(\theta_r).Rx(\lambda_r)` .
```

Comment:
These dots are not needed (same holds for others below, and for `\cdot` in between the gates, in case you'd like to change it) 🙂 
```suggestion
        :math:`K2_l = Ry(\theta_l) Rx(\lambda_l)` ,
        :math:`K2_r = Ry(\theta_r) Rx(\lambda_r)` .
```